pkg/api/unversioned/group_version.go

```diff
@@ -29,8 +29,8 @@ import (
 // `resource.group.com` -> `group=com, version=group, resource=resource` and `group=group.com, resource=resource`
 func ParseResourceArg(arg string) (*GroupVersionResource, GroupResource) {
 	var gvr *GroupVersionResource
-	s := strings.SplitN(arg, ".", 3)
-	if len(s) == 3 {
+	if strings.Count(arg, ".") == 3 {
+		s := strings.SplitN(arg, ".", 3)
```

Comment:
The identation here is wrong, but maybe this is exactly the same problem as we had before @smarterclayton ?
[The problem was that we were seeing two different things.]